R/executeDqChecks.R

```diff
@@ -13,8 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
+library(stringr)
```

Comment:
Avoiding use of `library` or `require` is [required](https://ohdsi.github.io/Hades/developerGuidelines.html#Function_calls_must_not_have_invisible_side_effects) as this will override any definitions in the user space.
Use namespace `stringr::<function>` instead